src/screens/SendToken/SendTokenAmount.js

```diff
@@ -151,7 +154,10 @@ class SendTokenAmount extends React.Component<Props, State> {
     this.state = {
       value: null,
       showModal: false,
+      gasLimit: 0,
     };
+
+    this.handleChange = debounce(this.handleChange, 500);
```

Comment:
You need to debounce the `getGasLimit()` method, not the handleChange

Reply:
Fixed in newer commits